scripts/themesConfig.js

```diff
@@ -22,6 +22,14 @@ try {
 const uuid = require('uuid');
 
 const hostFqdn = fqdn ? "http://" + String(fqdn()) : "";
+const auth = process.env["QWC2_WMS_HTTP_AUTH"] || null;
```

Comment:
You don't need `|| null`, do you?

Reply:
Not really indeed. A weird reflex of always initializing stuff and because I tend to consider there's a semantic difference between `null` and `undefined`... but it's a bit useless here and it works only if this convention is followed everywhere in the project.

Reply:
Agree :100:%